docs/build.md

```diff
@@ -0,0 +1,49 @@
+# Building a go binary that includes golang-fips patches
+
+## Env variables
+
+`GOEXPERIMENT=strictfipsruntime` helps ensure a fips compliant binary is built. Note that this is functionally equivalent of
+
+```
+GO_BUILDTAGS+="goexperiment.strictfipsruntime"
+```
+
+## Native build - Method 1
+
+The easiest way to do is to simply clone this repo, apply patches and from the internal go/src directory run ./make.bash. See a more comprehensive script at `scripts/build1.sh`
+
+```
+git clone https://github.com/golang-fips/go.git golang-fips
+cd golang-fips
+./scripts/full-initialize-repo.sh go$GOLANG_VER
```

Comment:
The default version described in go/config/versions.json is the only explicitly compatible upstream version at a given time.  It would be preferable to recommend calling this script without the version-override argument and just use the default configuration.